doc/source/whatsnew/v1.1.0.rst

```diff
@@ -960,6 +960,7 @@ Reshaping
 - Bug in :func:`concat` was not allowing for concatenation of ``DataFrame`` and ``Series`` with duplicate keys (:issue:`33654`)
 - Bug in :func:`cut` raised an error when non-unique labels (:issue:`33141`)
 - Ensure only named functions can be used in :func:`eval()` (:issue:`32460`)
+- Bug in :func:`aggregate` was causing recursive loop in some cases (:issue:`34224`)
```

Comment:
actually this is not right, can you put `` :func:`DataFrame.aggregate` `` and `` :func:`Series.aggregate` ``.

pls ping on green.